plugins/alarmclock/src/test/java/de/unistuttgart/iaas/amyassist/amy/plugin/alarmclock/AlarmClockResourceTest.java

```diff
@@ -85,7 +88,7 @@ void testGetAllAlarms() {
 			Timestamp expected = new Timestamp(alarms.get(i));
 			assertEquals(expected, ts[i]);
 			assertEquals(this.target.getUriBuilder().path(AlarmClockResource.class, "getAlarm").build(i).toString(),
-					ts[i].getLink());
+					ts[i].getLink().toString());
```

Comment:
the `toString` is not needed, both sides are URIs

Reply:
I just took that from the dev branch when merging